src/test/__tests__/ReactTestUtils-test.js

```diff
@@ -19,286 +22,290 @@ var ReactTestUtils;
 
 describe('ReactTestUtils', () => {
   beforeEach(() => {
+    createRenderer = require('ReactShallowRenderer').createRenderer;
     PropTypes = require('prop-types');
     React = require('react');
     ReactDOM = require('react-dom');
     ReactDOMServer = require('react-dom/server');
     ReactTestUtils = require('ReactTestUtils');
   });
 
-  it('should have shallow rendering', () => {
-    class SomeComponent extends React.Component {
-      render() {
+  // Shallow renderer only implemented for Fiber in 16+
+  if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
> Otherwise it looks like we stopped testing shallow renderer in Stack at all.

This new shallow renderer _is_ fiber only. Seemed like too much effort to create a new one that supports stack and works with our new bundles- just to throw it away. (I mentioned this in the description but maybe it wasn't prominent enough.)

I'm not sure how to address www yet.